packages/nextjs-mf/README.md

```diff
@@ -144,6 +145,20 @@ const SomeHook = require('next2/someHook');
 import SomeComponent from 'next2/someComponent';
 ```
 
+## Usage in Next.js App Router
+
+In order to use module federation in projects with app router, you must pass `next/navigation` as a shared dependency. This will disable sharing default dependencies (`DEFAULT_SHARE_SCOPE`).
+
+```js
+new NextFederationPlugin({
+  name: '',
+  filename: '',
+  remotes: {},
+  exposes: {},
+  shared: ['next/navigation'], // This is important!
```

Comment:
Add this to internal.ts instead, where Next.js retrieves its default shares. Users do not need to add extra shared module to make it work

Reply:
The reason I decided to use this was to ensure that `defaultShared` is not pased in app router. That creates issues with react-dom and react runtime.

It might be a better idea to add a new flag `enableAppRouter` in federation config - what do you think?

Reply:
Okay got another idea - I'll try to figure out whether the project uses app router and do this implicitly.

Reply:
What happens when if both pages and app router are used? 

Will it break?

If so we can enable "appRouterOnly" mode so its either one or the other